backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/database/Schema.kt

```diff
@@ -14,13 +14,15 @@ import org.jetbrains.exposed.sql.or
 
 const val CARD_INFO_HASH_LENGTH = 32 // Using SHA256-HMAC
 const val TOTP_SECRET_LENGTH = 20
+const val ACTIVATION_SECRET_LENGTH = 30
 
 enum class CodeType {
     static,
     dynamic
 }
 
 object Cards : IntIdTable() {
+    val activationSecret = binary("activationSecret", ACTIVATION_SECRET_LENGTH).nullable()
```

Comment:
Probably we should name this 
```suggestion
    val activationSecretHash = binary("activationSecretHash", ACTIVATION_SECRET_LENGTH).nullable()
```